openshift/auth.config.yaml

```diff
@@ -50,7 +50,7 @@ objects:
     notapproved_redirect: ""
     keycloak.url: https://sso.openshift.io
     notification.serviceurl: ""
-    email.verify.url: https://prod-preview.openshift.io/_home
-    invitation.accepted.url: https://prod-preview.openshift.io/_home
+    email.verify.url: https://openshift.io/_redirects/_verifyEmail
```

Comment:
AFAIK, this file is just an example. If there's something to fix on prod and pre-prod, we'll have to open a HK to update the configmaps